docs/DeveloperGuide.md

```diff
@@ -514,7 +561,7 @@ Given below is an example usage scenario and how the mechanism behaves:
 
 #### 3.6.2 Design Considerations (**Get Edit Recipe**) <a id="362-design-consideration-get-edit-recipe"></a>
 ##### Aspect: Concern while adding a new feature <a id="3621-aspect"></a>
-* Workflow must be consistent with other get edit commands.
+* Workflow must be consistent with other get edit commands e.g get ingredient command.
```

Comment:
I feel the eg is a bit extra cause we only have another one get edit command

Reply:
o ya ok